gio/src/auto/tls_certificate.rs

```diff
@@ -115,6 +138,31 @@ impl TlsCertificate {
         }
     }
 
+    #[cfg(any(feature = "v2_72", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_72")))]
+    #[doc(alias = "g_tls_certificate_new_from_pkcs12")]
+    #[doc(alias = "new_from_pkcs12")]
+    pub fn from_pkcs12(
+        data: u8,
+        length: usize,
```

Comment:
Missing array length annotation

Reply:
https://gitlab.gnome.org/GNOME/glib/-/merge_requests/2453

Reply:
Once that is merged I'll get `gobject-introspection` updated and then we can update the gir files another time.

Reply:
At least from now on we can notice such breakages as early as possible... Hoping to see g-i as part of glib in the future to no longer have to deal with this kind of issues.

Reply:
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/311